src/app/vector/qgsvectorelevationpropertieswidget.cpp

```diff
@@ -105,6 +105,8 @@ QgsVectorElevationPropertiesWidget::QgsVectorElevationPropertiesWidget( QgsVecto
   connect( mSurfaceLineStyleButton, &QgsSymbolButton::changed, this, &QgsVectorElevationPropertiesWidget::onChanged );
   connect( mSurfaceMarkerStyleButton, &QgsSymbolButton::changed, this, &QgsVectorElevationPropertiesWidget::onChanged );
   connect( mExtrusionGroupBox, &QGroupBox::toggled, this, &QgsVectorElevationPropertiesWidget::toggleSymbolWidgets );
+
+  setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#elevation-properties" ) );
```

Comment:
This one is still a PR. @nyalldawson can I have your feedback on https://github.com/qgis/QGIS-Documentation/pull/7698 and https://github.com/qgis/QGIS-Documentation/pull/7702 to make it available? Thanks.

Reply:
Thanks Nyall for the review. All the links in this PR will be operational tonight (CET) so IMHO it is good to merge.